go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/Jeffail/gabs/v2 v2.6.1
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/blang/semver v3.5.1+incompatible
+	github.com/docker/docker v20.10.21+incompatible
```

Comment:
i feel like we need a way to do these sorts of dependencies without checking them in. or we need a whole new project

Reply:
Yeah, I left a comment, I'm not sure how changing dependencies is managed so I would like to learn about that process.